api/handlers/listhandler.py

```diff
@@ -797,7 +797,7 @@ def download(self, cont_name, list_name, **kwargs):
                 ticket = util.download_ticket(self.request.client_addr, 'file', _id, filename, total_size, origin=self.origin)
             else:
                 targets, total_size, file_cnt = self._prepare_batch(fileinfo)
-                filename = 'analysis_' + analysis_id + '.tar'
+                filename = 'analysis_' + storage.get_analysis_label(_id, analysis_id) + '.tar'
```

Comment:
We'll want to make sure the file name we create is a reasonable file name that file systems can handle. I think we have a util function that strips them of all special characters already, I'll take a look.